src/sentry/api/bases/organizationmember.py

```diff
@@ -10,18 +10,38 @@
 from sentry.db.models.fields.bounded import BoundedAutoField
 from sentry.models.organization import Organization
 from sentry.models.organizationmember import InviteStatus, OrganizationMember
+from sentry.organizations.services.organization.model import (
+    RpcOrganization,
+    RpcUserOrganizationContext,
+)
 
 from .organization import OrganizationEndpoint, OrganizationPermission
 
 
 class MemberPermission(OrganizationPermission):
     scope_map = {
         "GET": ["member:read", "member:write", "member:admin"],
-        "POST": ["member:write", "member:admin"],
+        "POST": ["member:write", "member:admin", "member:invite"],
         "PUT": ["member:write", "member:admin"],
         "DELETE": ["member:admin"],
     }
 
+    def has_object_permission(
```

Comment:
What is this doing?

Reply:
has_object_permission runs inside `convert_args`, and does the endpoint scope checks for us. We need extra logic tho to check for the org bitflag on the POST.